packages/config/src/config/index.js

```diff
@@ -10,7 +10,7 @@ import router from './router'
 import server from './server'
 import cli from './cli'
 
-export const defaultNuxtConfigFile = `nuxt.config${process.env.NUXT_TS === 'true' ? '.ts' : '.js'}`
+export const defaultNuxtConfigFile = 'nuxt.config.js'
```

Comment:
Is `.js` necessary here?

Reply:
The main issue not having explictly the extension is for the watch of the config file

Reply:
Solution mentioned in other comment